shopBuddy/shopBuddy/ItemDetailViewController.m

```diff
@@ -21,12 +22,37 @@ @implementation ItemDetailViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    self.titleLabel.text = self.item.name;
-    self.brandLabel.text = self.item.brand;
-    self.descriptionLabel.text = self.item.item_description;
-    NSString *URLString = self.item.images[0];
-    NSURL *url = [NSURL URLWithString:URLString];
-    [self.itemImage setImageWithURL:url];
+    [[APIManager shared] getItem:^(Item *item, NSError *error)
+     {
+        NSLog(@"😎😎😎 Successfully loaded itemDetails");
+        if (item) {
+            NSLog(@"😎😎😎 Successfully loaded itemDetails");
+            self.item = item;
+            self.titleLabel.text = self.item.name;
+            self.brandLabel.text = self.item.brand;
+            self.descriptionLabel.text = self.item.item_description;
+            NSString *URLString = self.item.images[0];
+            NSURL *url = [NSURL URLWithString:URLString];
+            [self.itemImage setImageWithURL:url];
+            
+        }
+        else {
+            NSLog(@"😫😫😫 Error getting item details: %@", error.localizedDescription);
+        }
+    }];
+    /*
+    [[APIManager shared] getItemWithBarcode:self.barcode completion:^(NSDictionary *itemDetails, NSError *error) {
+        NSLog(@"😎😎😎 Successfully loaded itemDetails");
+        if (itemDetails) {
+            NSLog(@"😎😎😎 Successfully loaded itemDetails");
+            self.item = [[Item alloc] initWithDictionary:itemDetails[@"products"][0]];
+            
+        }
+        else {
+            NSLog(@"😫😫😫 Error getting item details: %@", error.localizedDescription);
+        }
+    }];
+     */
```

Comment:
Do not promote commented code to prod/master. You should have used `getItemWithBarcode` instead.